tests/unit/test_client.py

```diff
@@ -83,3 +92,27 @@ def test_pub(self):
                 self.assertRaises(SaltInvocationError,
                                   self.client.pub,
                                   'non_existent_group', 'test.ping', tgt_type='nodegroup')
+
+    @skipIf(not salt.utils.is_windows(), 'Windows only test')
```

Comment:
We have tests that run the TCP transport in Jenkins. Perhaps this check should run on windows and those TCP tests as well?

https://jenkinsci.saltstack.com/job/2017.7/view/Transport/

Reply:
Maybe I should be more clear in the docs that this is dealing with ZeroMQ's (TCP vs ICP). Not to be confused with Salt's TCP transport. Does that change things?

Reply:
Ah, yes. Thanks!

Reply:
I've added "ZeroMQ" to the doc strings in order to be more clear about what is being tested.